nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const app = express();
+
```

Comment:
I think you are missing here 
```suggestion
app.use(
  express.urlencoded({
    extended: true,
  })
);
app.use(express.json());
```
Have you tested POST requests with Postman?